app/Livewire/Project/New/DockerImage.php

```
@@ -26,12 +30,41 @@ public function mount()
+
+            return;
+        }
+
+        // Build the full Docker image string
+        if ($this->imageSha256) {
+            $dockerImage = $this->imageName.'@sha256:'.$this->imageSha256;
+        } elseif ($this->imageTag) {
+            $dockerImage = $this->imageName.':'.$this->imageTag;
+        } else {
+            $dockerImage = $this->imageName.':latest';
+        }
```

PHPMD (2.15.0)

The method submit uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)